api/src/main/java/jakarta/json/bind/annotation/JsonbNumberFormat.java

```diff
@@ -20,6 +20,7 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import java.util.Locale;
```

Comment:
Instead of importing for javadoc only, FQCN like in lines 54, 66 shall be used perhaps.